domains/raphael.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "raphael-as",
+        "email": "raphaelas.pro@gmail.com"
+    },
+    "record": {
+        "A": ["185.199.108.153"]
```

Comment:
I cannot reach your website (404)

if you want to use github pages it's 4 IPs
- `185.199.108.153`
- `185.199.109.153`
- `185.199.110.153`
- `185.199.111.153`

Reply:
yes i want to link to my website : [https://raphael-as.github.io](https://raphael-as.github.io)